src/mig/modules/example/example.go

```diff
@@ -48,6 +52,11 @@ func init() {
 type Runner struct {
 	Parameters params
 	Results    modules.Result
+
+	// dependency injections
+	hostnameFn       func() (string, error)
+	interfaceAddrsFn func() ([]net.Addr, error)
+	lookupHostFn     func(name string) (addrs []string, err error)
```

Comment:
These are where I inject the dependencies.  So in normal operation, `lookupHostFn` points to `net.LookupHost`, but in tests, it points to a fake.